pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -10937,6 +10937,7 @@
   hoc_events:
     heading: "Register to host an Hour of Code"
     desc: "Hour of Code is available year-round, but every year in December your class can join millions of students around the world celebrating Computer Science Education Week with the Hour of Code. Registration for the annual celebration starts each year in October."
+    desc_sept: "Hour of Code is available year-round, but every year in December your class can join millions of students around the world celebrating Computer Science Education Week with the Hour of Code. Registration for the annual celebration starts each year in September."
```

Comment:
Is it better to have the international version be in English or for the international version to say October do we think?

Reply:
I think we should keep it as September since the event form is going to go live for everyone at the same time.

Reply:
There is only 1 week between when its live and when its October. I think it may be worse to have it english but I would defer to global on that call

Reply:
Ah you're right. I'll post this in the #i18n channel.